docs/Images/Image.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Images | Merakii-UI</title>
+    <link rel="stylesheet" href="/CSS/style.css">
+</head>
+<body>
+    <div class="container">
+        <nav class="navbar">
+            <div class="header">
+                <a href="../../index.html" class="nav-header">Merakii UI</a>
+            </div>
+            <ul class="nav-links">
+                <li>
+                    <a href="../intro.html">Docs</a>
+                </li>
+                <li>
+                    <a href="https://github.com/thesudhansupanda/Merakii-UI">Github</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="menu-div">
+            <button class="btn btn-primary btn-float" id="menu-btn">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
```

Comment:
Check this. Are you deliberately trying to put just one button in this menu div?

Reply:
yes , its for mobile view !